### ISSUE TITLE
[Color Schemes]: Screen reader does not announce the 'default' visible text defined with the 'Campbell' color scheme on the page.

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/terminal#14401]
### Windows Terminal version

1.17.3052.0

### Windows build number

Windows 11 Version 22H2 (OS Build 25206.1000)

### Steps to reproduce

1. Open Windows Terminal.
2. Open 'Settings > Startup' window with 'Ctrl+,' keys.
3. Turn on Narrator by 'Ctrl+Win+Enter' keys.
4. Navigate to 'Color schemes' list item by down arrow key and select it.
5. Navigate to 'Campbel' by the tab key.
6. Observe the Narrator's announcement here.

**User Experience: -**
Screen reader users will have to navigate in scan mode to get this value information defined with the color control.

**Guideline Reference:**
(https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Attachments:**
[Screen reader does not announce the 'default' visible text defined with the 'Campbell' color scheme on the page..zip](https://github.com/microsoft/terminal/files/10028996/Screen.reader.does.not.announce.the.default.visible.text.defined.with.the.Campbell.color.scheme.on.the.page.zip)

### Expected Behavior

Screen reader should announce the 'default' visible text defined with the 'Campbell' color scheme on the page. It should announce as '_Campbell, default, 1 of 11 selected_'.

### Actual Behavior

Screen reader does not announce the 'default' visible text defined with the 'Campbell' color scheme on the page. It announces as '_Campbell, 1 of 11 selected_'.

**Observation:**

Issue also reproes with the NVDA screen reader.

**Comment by zadjii-msft** [comment]
FWIW, this still repros on the current Color Schemes page, circa 1.17.3481.0

**Comment by DHowett** [comment]
Reverted due to a virtualization-related crash in 7d0baa794/ab79a8538.

**Comment by zadjii-msft** [comment]
@DHowett Should we reopen this then? Or just file a new bug to track the crash?

**Comment by DHowett** [comment]
Ha, I actually did mean to reopen it. Thanks!
